CONTRIBUTING.md

```diff
@@ -1,3 +1,3 @@
-GitHub Issues is for reporting bugs in **<PODNAME>** and discussing features, bugs and general feedback. Be sure to check our [documentation](http://cocoadocs.org/docsets/<PODNAME>), [FAQ](https://github.com/hyperoslo/<PODNAME>/wiki/FAQ) and [past issues](https://github.com/hyperoslo/<PODNAME>/issues?state=closed) before opening any new issues.
+GitHub Issues is for reporting bugs in **<PODNAME>** and discussing features, bugs and general feedback. Be sure to check our [documentation](http://cocoadocs.org/docsets/<PODNAME>), [FAQ](https://github.com/<USERNAME>/<PODNAME>/wiki/FAQ) and [past issues](https://github.com/<USERNAME>/<PODNAME>/issues?state=closed) before opening any new issues.
```

Comment:
I personally like creating external links like this:

```
Blah blah blah [documentation] blah blah [FAQ] blah [past issues] and all that.

[documentation]: http://cocoadocs.org/docsets/<PODNAME>
[FAQ]: https://github.com/<USERNAME>/<PODNAME>/wiki/FAQ
[past issues]: https://github.com/<USERNAME>/<PODNAME>/issues?state=closed
```

I think it reads for much cleaner inline text and the result is the same. :smiley: 

> Blah blah blah [documentation] blah blah [FAQ] blah [past issues] and all that.

[documentation]: http://cocoadocs.org/docsets/<PODNAME>
[FAQ]: https://github.com/<USERNAME>/<PODNAME>/wiki/FAQ
[past issues]: https://github.com/<USERNAME>/<PODNAME>/issues?state=closed